integration_tests/pages/overview/TimelinePage.ts

```diff
@@ -35,25 +35,10 @@ export default class TimelinePage extends Page {
     return this
   }
 
-  hasLearningAndWorkProgressPlanEventWithOneGoalDisplayed(): TimelinePage {
-    // The timeline is rendered in reverse chronological order, so the last event on the timeline (ie. the last element in the dom)
-    // is the first event in time.
-    // We expect the last event in the DOM to be the first event in time, which should be ACTION_PLAN_CREATED
-    this.lastTimelineEvent().should('have.attr', 'data-qa-event-type', 'ACTION_PLAN_CREATED')
-    // We expect the 2nd to last event in the DOM to be the 2nd event in time, which should be GOAL_CREATED ('goal' singular)
-    this.lastButOneTimelineEvent().should('have.attr', 'data-qa-event-type', 'GOAL_CREATED')
-    return this
-  }
-
-  hasLearningAndWorkProgressPlanEventWithMultipleGoalsDisplayed(): TimelinePage {
-    // The timeline is rendered in reverse chronological order, so the last event on the timeline (ie. the last element in the dom)
-    // is the first event in time.
-    // We expect the last event in the DOM to be the first event in time, which should be ACTION_PLAN_CREATED
-    this.lastTimelineEvent().should('have.attr', 'data-qa-event-type', 'ACTION_PLAN_CREATED')
-    // We expect the 2nd to last event in the DOM to be the 2nd event in time, which should be MULTIPLE_GOALS_CREATED
-    this.lastButOneTimelineEvent().should('have.attr', 'data-qa-event-type', 'MULTIPLE_GOALS_CREATED')
-    return this
-  }
+  hasTimelineEventsInOrder = (events: string[]) =>
+    cy.get('div.moj-timeline div.moj-timeline__item').each((el, idx) => {
+      cy.wrap(el.attr('data-qa-event-type')).should('eq', events[idx])
```

Comment:
Credit to @nathanrussell-moj-digital for this. Not my area of expertise! 🤯

Reply:
Too kind! It's not my area of 'expertise' either. I might have had a little more exposure to things like the browser DOM (I wouldn't call that 'expertise' 😁 ), but the cypress API that encompasses that is mainly voodoo to me as well! 🤣 

In other news , the only thing I'd perhaps do to this method is declare a return type of `TimelinePage`, and then do `return this` at the end.
That way we can chain these method calls 👍

Reply:
Ah yes, good point. I had that in my original function but forgot to add it to this one!